### ISSUE TITLE
Codeblocks look weird

### ISSUE THREAD

**Issue by solumos** [BTV-Python/btv-python.github.io#6]
![capture](https://cloud.githubusercontent.com/assets/8253064/7492585/4ae8fb3e-f3c7-11e4-9dc8-17b2c24759dd.PNG)

**Comment by zevav** [comment]
agreed, it's weird.  mostly it's weird because of the white-edged box, right?

**Comment by solumos** [comment]
@zevav yeah, it's that box thing.
